demo-shell/src/app/components/cloud/service-task-list-cloud-demo.component.html

```diff
@@ -0,0 +1,29 @@
+<div fxLayout="column" fxFill fxLayoutGap="2px">
```

Comment:
maybe you can just use css flex instead of fx-* commands? that will help us get rid of the @angular/flex-layout dependency

Reply:
This is only at a demo-shell level, do you wanna get rid of it either way? we have it in other places too.

Reply:
for demo shell is fine like it is I think